src/test/remote/test_remote.py

```diff
@@ -0,0 +1,54 @@
+import yaml
+from subprocess import run, PIPE
+from unittest import TestCase
+from getpass import getuser
+from typing import Dict
+
+from ja.common.proxy.command_handler import CommandHandler
+
+
+class CommandHandlerDummy(CommandHandler):
+
+    def _process_command_string(
+            self, command_dict: Dict[str, object], type_name: str, username: str) -> Dict[str, object]:
+        assert username == getuser()
+        assert type_name == "COMMAND"
+        command_string = str(command_dict["payload"])
+        response_string = ""
+        while command_string.startswith("COMMAND"):
+            command_string = command_string[7:]
+            response_string += "RESPONSE"
+        return dict(payload=response_string)
+
+
+class RemoteTest(TestCase):
+    """
+    Class for testing Remote and CommandHandler.
+    IMPORTANT: THESE TESTS WILL FAIL IF YOU DON'T RUN THEM FROM THE BASE SRC DIRECTORY.
```

Comment:
Would this be true if PYTHONPATH is set appropriately??

Reply:
Yes, because PYTHONPATH determines where to look for modules only.
PYTHONPATH does **not** change the working directory of the python process.

Reply:
Wait, I think I understand what you mean.
I'll see if it works.

Reply:
I pushed a fixup.
Thanks for the pointer.